frontend/src/metabase/query_builder/components/dataref/ConnectedTableList.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { jt } from "ttag";
+
+import Table from "metabase-lib/lib/metadata/Table";
+
+import {
+  NodeListItem,
+  NodeListItemName,
+  NodeListItemIcon,
+  NodeListTitle,
+  NodeListContainer,
+  NodeListIcon,
+  NodeListTitleText,
+} from "./NodeList.styled";
+
+type Props = {
+  tables: Table[];
+  onTableClick: (table: Table) => void;
+};
+
+function ConnectedTableList({ tables, onTableClick }: Props) {
+  return (
+    <NodeListContainer>
+      <NodeListTitle>
+        <NodeListIcon name="connections" size="14" />
+        <NodeListTitleText>{jt`${tables.length} connections`}</NodeListTitleText>
```

Comment:
Regular `t` function should work here because there are no JSX parameters in the string